docs/content/src/learn/tutorials/saas-integrations/googlespreadsheet/backup-csv-files-using-googlespreadsheets/src/backup_csv_files/uploader.bal

```diff
@@ -0,0 +1,143 @@
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import wso2/gsheets4;
+import ballerina/time;
+import ballerina/log;
+import ballerina/config;
+
+// Define a Spreadsheet client
+gsheets4:SpreadsheetConfiguration spreadsheetConfig = {
+    oAuthClientConfig: {
+        accessToken: config:getAsString("ACCESS_TOKEN"),
+        refreshConfig: {
+            clientId: config:getAsString("CLIENT_ID"),
+            clientSecret: config:getAsString("CLIENT_SECRET"),
+            refreshUrl: config:getAsString("REFRESH_URL"),
+            refreshToken: config:getAsString("REFRESH_TOKEN")
+        }
+    }
+};
+gsheets4:Client spreadsheetClient = new(spreadsheetConfig);
+
+
+public function main() {
+    string csvFilePath = "src/backup_csv_files/resources/people.csv";
+    string spreadSheetName = "People_Info";
+    string sheetName = getCurrentDateName();
+    string topLeftCell = "A1";
+
+    // Create a Readable CSV Channel
+    io:ReadableCSVChannel|error readableCsvChannelOrError = io:openReadableCsvFile(csvFilePath);
+    if (readableCsvChannelOrError is error) {
+        log:printError("An error occurred opening the Readable CSV Channel: ", err = readableCsvChannelOrError);
+        return;
+    }
+    io:ReadableCSVChannel readableCsvChannel = <io:ReadableCSVChannel>readableCsvChannelOrError;
+
+    // Store CSV data from file to a String array
+    string[][] csvData = getCsvFromFile(readableCsvChannel);
+
+    //Create a GoogleSheet using the Spreadsheet client
+    gsheets4:Spreadsheet spreadsheet = checkpanic spreadsheetClient->createSpreadsheet(spreadSheetName);
+    string spreadsheetId = checkpanic <@untainted>spreadsheet.getSpreadsheetId();
+
+    //Add a new sheet
+    _ = checkpanic spreadsheetClient->addNewSheet(spreadsheetId, sheetName);
+
+    // Backup CSV Data to the new sheet
+    boolean successful = checkpanic spreadsheetClient->setSheetValues(spreadsheetId,
+        sheetName, csvData, topLeftCell, findBottomRightCell(csvData));
+
+    if (successful) {
+        io:println("Successfully backedup the CSV file.");
+    }
+
+    // Close the Readable CSV Channel
+    closeReadableCSVChannel(readableCsvChannel);
+
+    // Get the values from the spreadsheet
+    string[][] spreadsheetData = checkpanic spreadsheetClient->getSheetValues(spreadsheetId, sheetName, topLeftCell,
+                                                findBottomRightCell(csvData));
+
+    displaySpreadsheetData(spreadsheetData);
+}
+
+// Function that iterates through the obtained data from the spreadsheet
+function displaySpreadsheetData(string[][] spreadsheetData) {
+    io:println("**************** CSV Entries ****************");
```

Comment:
Better to replace `io:println` with `log:printInfo` in all places.

Reply:
Replaced all instances of io:println with log:printInfo